docs/users/basics/gas.md

```diff
@@ -11,36 +11,150 @@ Learn about the differences between `Gas` and `Fees` in Ethereum and Cosmos. {sy
 - [Cosmos SDK Gas](https://docs.cosmos.network/main/basics/gas-fees.html) {prereq}
 - [Ethereum Gas](https://ethereum.org/en/developers/docs/gas/) {prereq}
 
-The concept of Gas represents the amount of computational effort required to execute specific operations on the state machine.
+## Basics
 
-Gas was created on Ethereum to disallow the EVM (Ethereum Virtual Machine) from running infinite
-loops by allocating a small amount of monetary value into the system. A unit of gas, usually in the
-form of a fraction of the native coin, is consumed for every operation on the EVM and requires a
-user to pay for these operations. These operations consist in state transitions such as sending a
-transaction or calling a contract.
+### 1. Why do Transactions Need Fees?
```

Comment:
remove enumeration, since you are explaining different concepts rather then listing steps to follow